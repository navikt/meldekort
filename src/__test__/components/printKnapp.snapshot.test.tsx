import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper, {
  setLocalesBeforeAll,
} from '../testSetup/providerWrapper';
import PrintKnapp from '../../app/components/print/printKnapp';

setLocalesBeforeAll();

it('PrintKnapp', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <PrintKnapp
        innholdRenderer={() => {
          return <span>Test</span>;
        }}
      />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
