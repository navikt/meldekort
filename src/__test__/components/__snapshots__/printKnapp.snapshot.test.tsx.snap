// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrintKnapp 1`] = `
<div
  className="navigasjonsknapp"
>
  <button
    className="knapp knapp--flat"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    <img
      alt="Skriv ut logo"
      className="printLogo"
      src="print.svg"
    />
    Skriv ut
  </button>
</div>
`;
